### PR TITLE
bugfix/schedule-checkbox

### PR DESCRIPTION
**Pull request by M-Rybalko**

### CODE REVIEW

**Review by Markusplay — APPROVED** [review]

**Comment by papiroskada** [comment]
qa approved

**Comment by AnastasiaLytvyn** [comment]
shedule table is reloaded after chechking and unchecking all type of subjects

File: src/components/pages/schedule-page/SchedulePage.tsx [hunk 1]
```diff
@@ -55,6 +55,12 @@ const SchedulePage: FC<SchedulePageProps> = ({ semester, groups }) => {
   ]);
 
   useInitialise(semester, groups);
+  useEffect(() => {
+    const getEvents = async () => {
+      await handleWeekChange();
+    };
```
Comment (Markusplay) [inline src/components/pages/schedule-page/SchedulePage.tsx:61]: extract out of useEffect

File: src/store/schedule/useSchedule.ts [hunk 2]
```diff
@@ -119,13 +119,15 @@ export const useSchedule = create<State & Action>((set, get) => {
...
-        get().isUsingSelective
-          ? await get().loadNext5Auth(startWeek)
-          : await get().loadNext5(startWeek);
+      if (get().isUsingSelective) {
+        await get().loadNext5Auth(startWeek);
+        console.log('auth');
+      } else {
+        await get().loadNext5(startWeek);
+        console.log('not auth');
+      }
```
Comment (Markusplay) [inline src/store/schedule/useSchedule.ts:130]: remove console logs

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Comment by papiroskada** [comment]
qa approved

**Review by AnastasiaLytvyn — APPROVED** [review]
qa approve

**Review by Markusplay — APPROVED** [review]

**Pull request merged**